pyuvsim/tests/test_simsetup.py

```diff
@@ -550,13 +529,22 @@ def test_param_select_redundant():
 
 @pytest.mark.parametrize('case', np.arange(6))
 def check_uvdata_keyword_init(case):
-    base_kwargs = dict(
-        antenna_layout_filepath=os.path.join(SIM_DATA_PATH, "test_config/triangle_bl_layout.csv"),
-        telescope_location=(-30.72152777777791, 21.428305555555557, 1073.0000000093132),
-        telescope_name="HERA", Nfreqs=10, start_freq=1e8, bandwidth=1e8, Ntimes=60,
-        integration_time=100.0, start_time=2458101.0, polarization_array=['xx'],
-        no_autos=True, write_files=False, run_check=True
-    )
+    base_kwargs = {
+        "antenna_layout_filepath": os.path.join(SIM_DATA_PATH,
```

Comment:
this line will be flagged by black if we ever do that.

Reply:
There are a number of black changes that should happen, but I'm inclined to wait for a break in the PR traffic for that. I did make an issue about it though.